internal/imodels/imodels.go

```diff
@@ -47,10 +60,28 @@ func (pkg *PackageInfo) Name() string {
 		return pkg.purlCache.Name
 	}
 
+	// --- Make specific patches to names as necessary ---
+	// Patch Go package to stdlib
 	if pkg.Ecosystem().Ecosystem == osvschema.EcosystemGo && pkg.Inventory.Name == "go" {
 		return "stdlib"
 	}
 
+	// Patch python package names to be normalized
+	if pkg.Ecosystem().Ecosystem == osvschema.EcosystemPyPI {
+		// per https://peps.python.org/pep-0503/#normalized-names
+		return strings.ToLower(cachedregexp.MustCompile(`[-_.]+`).ReplaceAllLiteralString(pkg.Inventory.Name, "-"))
```

Comment:
Should we add a TODO to remove this after the API query change is deployed to production?

Reply:
Yep good catch, I'll probably remove this entirely since I don't plan on merging this PR in until next week.

Reply:
Hmm it looks like there's more work to be done here: 
1. Offline scanning also needs to this normalization
2. The code for identifying fix available/not available also needs this change.
Added a TODO.